dashboards/hbase/metadata.yaml

```diff
@@ -15,3 +15,19 @@ sample_dashboards:
     related_integrations:
       - id: hbase
         platform: GCE
+  -
+    category: HBase
+    id: hbase-prometheus-overview
+    display_name: Hbase Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Region Servers, Dead Region Servers, Read Requests, and Write Requests."
+    related_integrations:
+      - id: hbase
+        platform: GKE
+  -
+    category: HBase
+    id: hbase-operations-prometheus-overview
+    display_name: Hbase Operations Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Slow Get Operations, Slow Append Operations, Slow Delete Operations, Slow Put Operations, Slow Get Operations, Get Operations 99 Percentile, Append Operations 99 Percentile, Delete Operations 99 Percentile, Put Operations 99 Percentile, Increment Operations 99 Percentile, and Replay Operations 99 Percentile."
```

Comment:
Similar comment here as in the README.md file.

Reply:
This is the correct exporter